sql/5_0_1-to-5_0_2_upgrade.sql

```diff
@@ -86,3 +86,533 @@
 --    behavior: can take a long time.
 
 UPDATE `background_services` SET `require_once`='/library/MedEx/MedEx_background.php' WHERE `name`='MedEx';
+
+#IfNotTable form_eye_base
+DROP TABLE IF EXISTS `form_eye_base`;
```

Comment:
Remove these DROP TABLE statements from the upgrade script. Then if this goes horribly wrong it will die rather than potentially remove data.